tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -390,13 +387,18 @@ final class TestingServer(
     val uri = abspath.toURI.toString
     val extension = PathIO.extension(abspath.toNIO)
     val text = abspath.readText
-    server
+    val f = server
       .didOpen(
         new DidOpenTextDocumentParams(
           new TextDocumentItem(uri, extension, 0, text)
         )
       )
       .asScala
+    f.onFailure {
+      case error =>
+        println(s"Failed to open file! ${filename} with error ${error}")
```

Comment:
Why do we need it? If a future returns a failure then the tests should fail with this error anyway. :thinking: